policyengine_us/variables/gov/irs/income/taxable_income/deductions/qualified_business_income_deduction/qualified_business_income_deduction.py

```diff
@@ -4,12 +4,24 @@
 class qualified_business_income_deduction(Variable):
     value_type = float
     entity = TaxUnit
-    label = "Qualified business income deduction"
+    label = "Qualified business income deduction for tax unit"
```

Comment:
Is this needed?

Reply:
I added "for tax units" because the new `qbid_amount` has the same label except is ends in "amount for a person".   So, the variable labels emphasis the difference between the two variables with one being defined a the Person level and the other being defined at the TaxUnit level.